{
 "PRed": [
  {
   "PR": {
    "labels": [],
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.9.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [
     {
      "name": "bot-rerun"
     }
    ],
    "number": 2,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1706444277.9242265,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python312"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [
     {
      "name": "bot-rerun"
     }
    ],
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1723216975.882742,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python312"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 4,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.10.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 5,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.11.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2024.08.12.13.47.31",
 "pre_pr_migrator_attempts": {
  "python312": 69
 },
 "pre_pr_migrator_status": {
  "python312": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/10438798230\">bot CI job</a>): main:\nError running check-solvable in container - return code 137:\ncmd: ['docker',\n 'run',\n '-e',\n 'CF_TICK_IN_CONTAINER=true',\n '-e',\n 'RUN_URL',\n '--security-opt=no-new-privileges',\n '--read-only',\n '--cap-drop=all',\n '--mount',\n 'type=tmpfs,destination=/tmp,tmpfs-mode=1777,tmpfs-size=6000000000',\n '-m',\n '6000m',\n '--cpus',\n '1',\n '--ulimit',\n 'nofile=1024:1024',\n '--ulimit',\n 'nproc=2048:2048',\n '--rm',\n '-i',\n '--mount',\n 'type=bind,source=/tmp/tmp7eigoz5j/hidefix-feedstock,destination=/cf_tick_dir,readonly',\n 'ghcr.io/regro/conda-forge-tick:2024.8.22',\n '/opt/conda/envs/cf-scripts/bin/python',\n '-u',\n '/opt/autotick-bot/docker/run_bot_task.py',\n 'check-solvable',\n '--timeout',\n '600',\n '--verbosity',\n '2',\n '--log-level',\n 'info']\noutput: ''"
 },
 "smithy_version": "3.38.0"
}